### DISCUSSION TITLE
Using `$this->travelTo` to change the time breaks file upload test

### DISCUSSION THREAD

**Discussion by cdarken** [livewire/livewire#8307] [7. Bugs (archived)] [answered]
### Livewire version

v3.4.10

### Laravel version

v11.3.1

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Create a component that processes uploaded files
2. Write a test for it use `UploadedFile::fake()->image()`, set the time to something in the future, like `$this->travelTo(now()->addMonth())`
3. Run the test, the validation fails with invalid image and invalid mime types.

### Repository or Wirebox URL

https://github.com/cdarken/livewire-testing-upload-bug

### Additional information

In the provided repository there are also two tests using a plain Laravel controller.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment, marked as answer]
Fix: https://github.com/livewire/livewire/pull/8328
